data-transfer-service/src/main/java/com/epam/pipeline/dts/transfer/service/impl/DtsSynchronizationService.java

```diff
@@ -17,8 +17,11 @@
 package com.epam.pipeline.dts.transfer.service.impl;
```

Comment:
What do you think about merging `DtsSynchronizationService` and `AutonomousSynchronizationService`? The distinction between them doesn't seem quiet obvious.